WordPress/src/main/java/org/wordpress/android/ui/accounts/SitesListAdapter.java

```diff
@@ -0,0 +1,315 @@
+package org.wordpress.android.ui.accounts;
```

Comment:
It is indeed a cut-down version of SitePickerAdapter. Just gave it another shot and yeah, makes sense to re-use the original with minor modification. Done that with b7fcc74. Have a look! Thanks!